### PR TITLE
railsgirls v1

### PR DESCRIPTION
**Pull request by prayagb**
**Add a separate page for rails girls #42**

Here is a v1 of the new Rails Girls page. Please review and suggest changes. It does not have much content bec I do not have much info.Please feel free to rip it apart and ask me to change as required.

I realise now that I should have named my branch railsgirlsnewpage or something like that but I have already pushed it and I am sure there are going to be changes, so here it is. I will name the next version appropriately and get rid of this one. I made my changes on top of the features/white-theme branch

Attached are screenshots
railsgirls_page1
railsgirls_page2
railsgirls_page3

### CODE REVIEW

File: source/rails-girls/index.haml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        - data.coaches.each do |coach|
+          .col-xs-6.col-md-3.portfolio-item
+            = link_to "https://twitter.com/#{coach.twitter}", class: 'portfolio-link', rel: 'nofollow' do
+              .text-center.small
+                %b= coach.name
+                .black
+                  @#{coach.twitter}
+
+
+:javascript
```
Comment (borisrorsvort) [inline source/rails-girls/index.haml]: I'd extract these to a partial for eventual reuse

File: source/rails-girls/index.haml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  !function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+'://platform.twitter.com/widgets.js';fjs.parentNode.insertBefore(js,fjs);}}(document, 'script', 'twitter-wjs');
+
+
+.fb-root
+  :javascript
```
Comment (borisrorsvort) [inline source/rails-girls/index.haml]: Same here

**Comment by borisrorsvort** [comment]
@prayagb 🎉 for the PR. Much appreciated! 

For the content You can take the one the railsgrils official page and reformulate it a bit (http://railsgirls.com/brussels)

Having content is mandatory to get this merged

Also you need need to rebase your branch against master. (first need to update your own master from the original repo => https://help.github.com/articles/syncing-a-fork/)

**Comment by borisrorsvort** [comment]
For the last screenshot, I'd keep the big title on one line to keep the consistency with the other pages.
Maybe add two pictures so you can make an extra section with the images (cf the brug page)

Thanks in advance

File: data/coaches.yml [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+-
+  name: Javier Sanz
+  description:
+  image:
+  twitter: malditojavi
+-
+  name: Laura Gaetano
+  description:
+  image:
+  twitter: alicetragedy
```
Comment (borisrorsvort) [inline data/coaches.yml]: You forgot about me ;) Also are old coaches, not sure about their presence for 2016. Maybe we shouldnt mention the date at all ^^

Comment (prayagb) [inline data/coaches.yml]: Apologies for not including you. I took the names from the railsgirls.com/brussels website and I notice now that Lien is also missing. Good idea to exclude the date and say 'The Team'. Perhaps you could validate the list of coaches and let me know who to add/remove. Thanks!

Comment (borisrorsvort) [inline data/coaches.yml]: No prob it was mre like a joke :) For me its doesnt really need to be validated, everybody who helped coach in the past can be on that list. We can always add some more later ;)

**Comment by prayagb** [comment]
This branch and PR has been replaced by #60

**Pull request closed without merging**